tools/get-owner.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/bash
+# get-owner displays the current owner of the proxy contract
+# Assumption: RPC_URL, PROXY_ADDRESS env vars are set
+# Assumption: forge, cast, jq are in the PATH
+#
+echo "Getting current owner of Payments contract"
+
+if [ -z "$RPC_URL" ]; then
+  echo "Error: RPC_URL is not set"
+  exit 1
+fi
+
+if [ -z "$PROXY_ADDRESS" ]; then
+  echo "Error: PROXY_ADDRESS is not set"
+  exit 1
+fi
+
+echo "Getting current owner of proxy at $PROXY_ADDRESS"
+CURRENT_OWNER=$(cast call --rpc-url "$RPC_URL" "$PROXY_ADDRESS" "owner()")
+
+echo ""
+echo "=== OWNER INFORMATION ==="
+echo "Proxy Address: $PROXY_ADDRESS"
```

Comment:
Note this is a confusing name.  The fact the payments contract address is a proxy of an implementation address is a detail users do not care about.  This is the stable address people will continue to see.  This issue doesn't matter too much because normal users won't be interacting with Makefiles much.  But its better practice to cal this Payments Contract Address.  When referring to the other address you can call that IMPLEMENTATION_ADDRESS.